modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,62 @@
 = Flak
 
+These units are specialised to shoot down enemy flyers and drop pods.
+Some use very sophisticated targeting devices and fast, accurate weapons systems.
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[IMPORTANT]
+.Flak and drop pods
+====
+{flak} units may fire on drop pods as if they were flyers, against the Armour value of the transported unit.
+====
+
+== Flak orders
```

Comment:
```suggestion
.Flak orders
```
Don't accept this suggestion, I'll just explain my thinking and we can refine the idea together:
I have used this 'label, not heading' approach in some other cases too. Usually where there are options or sub-sets that might not warrant a whole heading level to themselves in the navigation.
But then I tend to upgrade them to a heading if I think that people are very likely to find it useful to have a URL anchor at that point, for sharing on Facebook etcetera. Which could well make sense in this case, so your change might be best.

Reply:
OK, but if you're going to use labels instead of headings the CSS needs to be altered accordingly because the labels are visually _far_ too subtle. And as you say, in this case it's something where players are going to want to go be able to go straight to on a page.